UI/src/router.js

```diff
@@ -52,6 +52,16 @@
       });
     };
 
+    var pointCentering = function (layerName, id) {
+      applicationModel.selectLayer(layerName);
+      var asset = _(models.pointAssets).find({layerName: layerName});
+      backend.getPointAssetById(id, layerName).then(function (result) {
+        mapCenterAndZoom(result.lon, result.lat, 12);
+
+        asset.selectedLinearAsset.open(result);
```

Comment:
`selectedLinearAsset` probably was a mistake, not a bad name, since the asset has the `selectedPointAsset` property. So I suggest to not change the name for something more generic, and just modify the `selectedLinearAsset` with `selectPointAsset `